README.md

```diff
@@ -0,0 +1,27 @@
+# One Login - Android
+
+This repository hosts the Android Mobile app for GOV.uk's One Login.
+
+## Branching strategy and releases
+
+This project uses a git-flow branching strategy, briefly:
+- Create a new branch from `develop`
+- Merge pull requests into `develop`
+- At the point of preparing a new release create a `release/` branch from the specified commit in develop
+- Name the branch using the appropriate semantic version for the release; for example `release/1.0.0`
+  - The initial version number for the release is taken from the branch name; `v1.0.0`
+  - Any subsequent pushes to the branch will increment the patch; `v1.0.1` 
+- Any required fixes for the release should be pull requested into the release branch
+- Once approval has been granted for the release to be published the release branch should be merged into `main`
+- `main` should then be merged back into `develop`
+
+## Getting started
+
+A number of how-to documents are provided to get a new developer up to speed with:
+- the required software
```

Comment:
@cgedrim-co I don't think we should include all these bullets here if we plan to merge this PR with only `requiredSoftware.md` - this paragraph will be misleading. (Small PRs are absolutely fine, but they need to stand alone in case we need to pivot or indeed run out of time within the sprint.)

Also please could you note the description on the ticket? 

I'm really keen that we (quickly) focus on enabling other developers to:
(a) run the code
(b) run the tests

Anything else - eg. extending functionality of a class and ensuring code is clean - is a bit beyond the scope at this stage. (I'm not saying it's not important, it's just not our very highest priority thing right now.)